assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Pagination.js

```diff
@@ -109,7 +115,7 @@ const Pagination = ( { tab } ) => {
 };
 
 Pagination.propTypes = {
-	tab: PropTypes.string,
+	tab: PropTypes.oneOf( [ 'new-ideas', 'saved-ideas', 'draft-ideas' ] ),
```

Comment:
I'm surprised these aren't constants defined somewhere :smile: 

We could also define `PropTypes.oneOf( [ 'new-ideas', 'saved-ideas', 'draft-ideas' ] )` in a constants file somewhere.  `react/prop-types` gets confused by this but we're not using that anywhere